src/openfluid/ui/waresdev/GitImportWorker.cpp

```diff
@@ -121,4 +123,22 @@ bool GitImportWorker::runImports()
 }
 
 
+// =====================================================================
+// =====================================================================
+
+
+void GitImportWorker::checkoutCurrentOpenFLUIDBranch(const QString& Path)
+{
+  GitUIProxy Git;
+  if (Git.checkout(Path, GitUIProxy::getCurrentOpenFLUIDBranchName()))
+  {
+    emit info(tr("Successful checkout of the current OpenFLUID version branch"));
+  }
+  else
+  {
+    emit error(tr("Unable to checkout branch corresponding to current OpenFLUID version branch."));
```

Comment:
This should be marked as "warning" (written using orange in the dialog) as it is not a fatal error for the whole process